### PR TITLE
Adding GreenLightning

### PR DESCRIPTION
**Pull request by nathantippy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,6 +46,7 @@ env:
      - "TESTDIR=Java/curacao"
      - "TESTDIR=Java/dropwizard"
      - "TESTDIR=Java/gemini"
+     - "TESTDIR=Java/greenlightning
```
Comment (nbrady-techempower) [inline .travis.yml]: Missing the closing quotation mark here, so this failed to run on travis.

File: frameworks/Java/greenlightning/src/main/java/com/ociweb/gl/benchmark/JSONBehaviorInstance.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import com.ociweb.pronghorn.network.config.HTTPContentTypeDefaults;
+
+public class JSONBehaviorInstance implements RestListener {
+
+	byte[] messagePayload;
+
+	private static final JSONRenderer<JSONBehaviorInstance> renderJSON = new JSONRenderer<JSONBehaviorInstance>()
+			.startObject()
+				.string("message", (o,t) -> t.write(o.messagePayload) )
+			.endObject();
```
Comment (nbrady-techempower) [inline frameworks/Java/greenlightning/src/main/java/com/ociweb/gl/benchmark/JSONBehaviorInstance.java]: https://frameworkbenchmarks.readthedocs.io/en/latest/Project-Information/Framework-Tests/#json-serialization Instantiation of the object that's going to be serialized has to happen on every request.

File: .travis.yml (hunk 1 above)
Comment (nathantippy) [reply .travis.yml]: Drat, my bad, will fix tonight.

File: frameworks/Java/greenlightning/src/main/java/com/ociweb/gl/benchmark/JSONBehaviorInstance.java (hunk 2 above)
Comment (nathantippy) [reply frameworks/Java/greenlightning/src/main/java/com/ociweb/gl/benchmark/JSONBehaviorInstance.java]: I am adding a temp object and will resubmit

**Comment by nathantippy** [comment]
will resubmit before morning.

**Pull request closed without merging**